scenario/migrate_resources.yaml

```diff
@@ -0,0 +1,51 @@
+namespace:
```

Comment:
Please add comments or docstring describing the purpose of this scenario and when to use it